skt/executable.py

```diff
@@ -912,5 +913,5 @@ def main():
         main()
     except KeyboardInterrupt:
         # cleanup??
-        print("\nExited at user request.")
+        print"\nExited at user request."
```

Comment:
I thought we switched to `print_function` in skt already. Seems not. Can you use that instead?